loader.lua

```diff
@@ -83,6 +84,7 @@ forceLanguage = false
 enableTokenButtons = false
 blowCache = false
 pngGraphics = true
+getFirstPrint = true
```

Comment:
Do you mind defaulting this to false?
```suggestion
getFirstPrint = false
```